### DISCUSSION TITLE
UST Transfer is stucked from Oasis Emerald to Terra

### DISCUSSION THREAD

**Discussion by Pousito** [wormhole-foundation/wormhole#981] [Problems / bugs] [answered]
Hi, I sent UST from Oasis to Terra but transfer is stuck more than 2 hours.

![Image1](https://user-images.githubusercontent.com/101228229/157348972-476b1a0c-eb87-468f-9533-a0fdbce82e2f.jpg)

When try to redeem this happen:

![image](https://user-images.githubusercontent.com/101228229/157349170-433f3b73-b2b5-4483-9403-d01a80a0a908.png)

Supports on telegram and discord have been informed of my sutuacion.

Oasis Esmerald transaction is: 

0xfe7bea9126ad7f5b8e8b2f3e4ebe7539e316c2cb444b1f159e321b9ba7350b5d

Thank you.

**Comment by evan-gray** [comment, marked as answer]
Since this post, this transaction has been redeemed.
